.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -25,6 +25,11 @@ body:
           required: true
         - label: I have resolved all warnings from `brew doctor` and that did not fix my problem.
           required: true
+        - label: I have not set `HOMEBREW_BOTTLE_DOMAIN`, `HOMEBREW_ARTIFACT_DOMAIN`, `HOMEBREW_BREW_GIT_REMOTE`, or `HOMEBREW_CORE_GIT_REMOTE` and am still able to reproduce this issue.
```

Comment:
@Homebrew/maintainers I'm debating whether we should just warn on these being set in `brew doctor`. That seems in keeping with the "you can do this but we don't really support it and want you to try without before you report it"?

Reply:
I’d approve of the idea. The fewer checkboxes there are in an issue, the likelier users will read them and not just tick blindly.

Reply:
+1 for this. So let's just add an entry on new issue page about how to report to public mirror sites, and remove this checkbox in bug report template?